File: src/main/java/spring/controllers/test/TestController.java
package spring.controllers.test;

import org.springframework.stereotype.Controller;
import org.springframework.ui.ModelMap;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class TestController {
    @GetMapping(path = "test")
    public String toTest(ModelMap model) {
        model.addAttribute("number", 2);
        model.addAttribute("test2", "test2");
        return "test/test";
    }
}

File: src/main/java/spring/controllers/test/TestController.kt
package spring.controllers.test

import judge.JudgeSystemConstant
import org.springframework.stereotype.Controller
import org.springframework.ui.ModelMap
import org.springframework.web.bind.annotation.GetMapping

@Controller
public class TestController {
    @GetMapping(path = ["test"])
    public fun toTest(modelMap: ModelMap): String {
        modelMap.addAttribute("test", JudgeSystemConstant.AC)
        return "test/test"
    }
}